lib/conversion.py

```diff
@@ -158,7 +158,25 @@ def lessons_question_correct(lessons_soup):
         if 'answers' in qAttr:
             for answer in qAttr['answers']:
                 if answer['correct']:
-                    return True;
+                    return True
+
+
+def lessons_hyperlinks(lessons_soup):
```

Comment:
As per above, rather than true / false and then another method to get detail, just a check that returns None or False or otherwise the detail.

Reply:
Updated, see ticket for details/visuals of the tables.